src/main/java/jp/classmethod/aws/gradle/cloudformation/AmazonCloudFormationMigrateStackTask.java

```diff
@@ -163,8 +163,8 @@ private void updateStack(AmazonCloudFormation cfn) throws IOException {
 		}
 		if (isCapabilityIam()) {
 			Capability selectedCapability =
-					(useCapabilityIam == null) ? Capability.CAPABILITY_IAM : useCapabilityIam;
-			getLogger().error("Using policy: " + selectedCapability);
+					(getUseCapabilityIam() == null) ? Capability.CAPABILITY_IAM : getUseCapabilityIam();
+			getLogger().info("Using IAM capability: " + selectedCapability);
```

Comment:
This line will conflict with the commit 90e74664ed37232a68c675b8cf35227bd5ecd6db in the pull request #89